src/custom/utils/signatures.ts

```diff
@@ -164,9 +164,10 @@ async function _signPayload(
       case 'v3':
         _signer = new TypedDataV3Signer(signer)
         break
-      case 'int_v4':
-        _signer = new IntChainIdTypedDataV4Signer(signer)
-        break
+      // TODO: This signer was remove from the SC npm package. Is it because this was a temporal fix because MM was broken in the past?
+      // case 'int_v4':
+      //   _signer = new IntChainIdTypedDataV4Signer(signer)
+      //   break
```

Comment:
@alfetopito @nlordell let me know if it's fine to just kill this code and my TODO comment is right

Reply:
This change might need to be forward-ported.

Reply:
https://github.com/gnosis/gp-v2-contracts/pull/768

Reply:
It might be no longer needed, need to check whether it works without it.
Still, foward-porting won't hurt

Reply:
Should be included in https://www.npmjs.com/package/@gnosis.pm/gp-v2-contracts/v/1.0.2.

Note that this fixes a `ethers` peer dependency (which was incorrectly set to `~5.0.18` but should have been `~5.4.0`.

Reply:
For me it worked without it, we would need to test all the mobile wallets again if we want to remove...

Reply:
To keep changes to the minimum, I will just use the signer since you also added it to 1.0.2

We can test with Walleth in a separate issue, I think removing this don't need to be done strictly now
90942384c44aa5d918ca617b66ef10a19f52cc94

Reply:
It might, but keep in mind WallETH is currently not supported :shrug: 
Anyway, we already have the code for it, won't hurt to keep it